pkg/operators/ebpf/converters.go

```diff
@@ -106,10 +112,63 @@ func (i *ebpfInstance) initEnumConverter(gadgetCtx operators.GadgetContext) erro
 	return nil
 }
 
+func (i *ebpfInstance) initStackConverter(gadgetCtx operators.GadgetContext) error {
+	kernelSymbolResolver, err := kallsyms.NewKAllSyms()
+	if err != nil {
+		return err
+	}
+	for _, ds := range gadgetCtx.GetDataSources() {
+		for _, in := range ds.GetFieldsWithTag(StackIdKernelType) {
+			if in == nil {
+				continue
+			}
+			if i.stackIdMap == nil {
+				return errors.New("kernel stack map is not initialized but used. " +
+					"if you are using `gadget_kernel_stack` as event field, " +
+					"try to include <gadget/kernel_stack_map.h>")
+			}
+			in.SetHidden(true, false)
+			out, err := ds.AddField(in.Name()+"_symbolized", api.Kind_String)
```

Comment:
@mauriciovasquezbernal @flyth Given the change in #3023, should we no longer add the "_symbolized" suffix?

(Not for this PR, since #3023 is not merged yet)